i18n/de_DE/mail/membership_application_confirmation/intro.twig

```diff
@@ -0,0 +1,5 @@
+herzlich willkommen bei Wikimedia Deutschland e.V.
```

Comment:
Comes after a "Sehr geehrtes Mitglied,\n" or the like so should be correct the way it is. Anyways, this is user-editable content.